chromium_src/chrome/browser/renderer_context_menu/render_view_context_menu.cc

```diff
@@ -634,6 +650,14 @@ void BraveRenderViewContextMenu::BuildAIChatMenu() {
 
   ai_chat_submenu_model_.AddTitleWithStringId(
       IDS_AI_CHAT_CONTEXT_QUICK_ACTIONS);
+
+#if BUILDFLAG(ENABLE_AI_REWRITER)
+  if (ai_rewriter::features::IsAIRewriterEnabled()) {
+    ai_chat_submenu_model_.AddItemWithStringId(IDC_AI_CHAT_CONTEXT_REWRITE,
+                                               IDS_AI_CHAT_CONTEXT_REWRITE);
```

Comment:
String for this item already present in the code - see https://github.com/brave/brave-core/commit/5e2aa2c52fbf7ce7a897f91deea82d6494ae18f1#diff-a379a32a9c1608092030ed9612501a57a39a4158de7c6e7e702b290a3a6e84a7R120